reana_workflow_controller/rest/utils.py

```diff
@@ -270,10 +270,16 @@ def delete_workflow(workflow, all_runs=False, workspace=False):
                 _mark_workflow_as_deleted_in_db(workflow)
                 remove_workflow_jobs_from_cache(workflow)
 
+            if all_runs:
+                message = "All workflows named {0} successfully deleted.".format(
+                    workflow.name
+                )
```

Comment:
minor: just as a note, this could be rewritten using f-strings, which tend to be more compact:
```python
message = f"All workflows named {workflow.name} successfully deleted."
```
(I am going to merge anyway, no need to change this!)